src/lang/extra/english.txt

```diff
@@ -195,6 +195,10 @@ STR_CONFIG_SETTING_REALISTIC_BRAKING_ASPECT_LIMITED_HELPTEXT    :In the realisti
 STR_CONFIG_SETTING_LIMIT_TRAIN_ACCELERATION                     :Limit train acceleration: {STRING2}
 STR_CONFIG_SETTING_LIMIT_TRAIN_ACCELERATION_HELPTEXT            :When using the realistic train braking model, also limit the maximum acceleration of trains. This is to prevent passengers from spilling their drinks due to some train types otherwise accelerating excessively quickly.
 
+STR_CONFIG_SETTING_TRAIN_ACC_BRAKING_PERCENT                     :Train acceleration/braking percent: {STRING2}
+STR_CONFIG_SETTING_TRAIN_ACC_BRAKING_PERCENT_HELPTEXT            :The percentage by which train acceleration and braking is adjusted.{}{}This setting can be used to alter the acceleration and braking of trains, without changing the maximum speed of trains.{}{}The default value is {NUM}.
```

Comment:
```suggestion
STR_CONFIG_SETTING_TRAIN_ACC_BRAKING_PERCENT                     :Train acceleration/braking percent: {STRING2}
STR_CONFIG_SETTING_TRAIN_ACC_BRAKING_PERCENT_HELPTEXT            :The percentage by which train acceleration and braking is adjusted.{}{}This setting can be used to alter the acceleration and braking of trains, without changing the maximum speed of trains.
```

There is no need to mention the default value in the setting description, as the default value is already mentioned in the settings window automatically for all settings.

A better name for the setting name would be "Train acceleration/braking scale factor"

Reply:
Resolved x2